src/main/java/use_case/recipeCollection/RecipeCollectionInputData.java

```diff
@@ -0,0 +1,8 @@
+package use_case.recipeCollection;
+
+/**
+ * Input Data for Recipe Collection use case.
+ */
+public class RecipeCollectionInputData {
+    // current user's Recipe Collection, so do not need extra input
```

Comment:
Great beginning structure for RecipeCollectionInputData! Feedback I would recommend is to create clearer and accurate comments so when you come back you understand where you left off.